configure.ac

```diff
@@ -1,5 +1,5 @@
 AC_PREREQ(2.59)
-AC_INIT([singularity],[2.4.5],[gmkurtzer@gmail.com])
+AC_INIT([singularity],[2.4.6rc2],[gmkurtzer@gmail.com])
```

Comment:
rpmbuild blows up with there's a '-' in the version. Seems to work fine without it.

Reply:
Stupid rmpbuild.  😕 Guess that means that 2.5.0-rc probably isn't building right now either.  I guess it didn't get run through Travis since I just pushed directly to the repo and then tagged?